### PR TITLE
Update IbftConfig Fields

### PR DESCRIPTION
**Pull request by rain-on**
The RequestTimeout field in the IbftConfig has been updated to be
"RequestTimeoutSeconds".

### CODE REVIEW

File: config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public int getBlockPeriodSeconds() {
     return ibftConfigRoot.getInteger("blockperiodseconds", DEFAULT_BLOCK_PERIOD_SECONDS);
   }
 
-  public int getRequestTimeoutMillis() {
-    return ibftConfigRoot.getInteger("requesttimeout", DEFAULT_ROUND_EXPIRY_MILLISECONDS);
+  public int getRequestTimeoutSeconds() {
+    return ibftConfigRoot.getInteger("requesttimeoutseconds", DEFAULT_ROUND_EXPIRY_MILLISECONDS);
```
Comment (Errorific) [inline config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java]: still got DEFAULT_ROUND_EXPIRY_MILLISECONDS while we're now talking about seconds

File: pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java [hunk 2]
```diff
@@ -187,7 +187,7 @@
 
     final IbftStateMachine ibftStateMachine = new IbftStateMachine(blockCreatorFactory);
     final IbftProcessor ibftProcessor =
-        new IbftProcessor(ibftEventQueue, ibftConfig.getRequestTimeoutMillis(), ibftStateMachine);
+        new IbftProcessor(ibftEventQueue, ibftConfig.getRequestTimeoutSeconds(), ibftStateMachine);
```
Comment (Errorific) [inline pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java]: Far side of this call is still referring to it's parameter as millis

File: config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java (hunk 1 above)
Comment (rain-on) [reply config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java]: oops. Done.

File: config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java [hunk 3]
```diff
@@ -36,7 +36,7 @@ public int getBlockPeriodSeconds() {
     return ibftConfigRoot.getInteger("blockperiodseconds", DEFAULT_BLOCK_PERIOD_SECONDS);
   }
 
-  public int getRequestTimeoutMillis() {
-    return ibftConfigRoot.getInteger("requesttimeout", DEFAULT_ROUND_EXPIRY_MILLISECONDS);
+  public int getRequestTimeoutSeconds() {
```
[the remark this answers is not in the document]
Comment (rain-on) [reply config/src/main/java/tech/pegasys/pantheon/config/IbftConfigOptions.java]: I agree, but can we worry about this one next, don't want to stray TOO far right now.

File: pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java (hunk 2 above)
Comment (rain-on) [reply pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java]: Done.

**Pull request merged**